src/pages/DataSharingLanguageTool.js

```diff
@@ -0,0 +1,232 @@
+        ]),
+        div({}, [
+          RadioButton({
+            value: 'general',
+            defaultChecked: general,
+            onClick: () => {
+              setGeneral(!general), setHmb(false), setDiseases(false), setOther(false);
+            },
+            label: 'General Research Use: ',
+            description: 'use is permitted for any research purpose',
+            sdsl: true
+          }),
+
+          RadioButton({
+            value: 'hmb',
+            defaultChecked: hmb,
+            onClick: () => {
+              setHmb(!hmb), setGeneral(false), setDiseases(false), setOther(false);
+            },
+            label: 'Health/Medical/Biomedical Use: ',
+            description: 'use is permitted for any health, medical, or biomedical purpose',
+            sdsl: true
+          }),
+
+          RadioButton({
+            value: 'diseases',
+            defaultChecked: diseases,
+            onClick: () => {
+              setDiseases(!diseases), setHmb(false), setGeneral(false), setOther(false);
+            },
+            label: 'Disease-related studies: ',
+            description: 'use is permitted for research on the specified disease',
+            sdsl: true
+          }),
+
+          div({
+            style: {buttonStyle, marginBottom: '10px', cursor: diseases ? 'pointer' : 'not-allowed'},
+          }, [
+            h(AsyncSelect, {
+              isDisabled: !diseases,
+              isMulti: true,
+              loadOptions: (query, callback) => searchOntologies(query, callback),
+              onChange: (option) => option ? setOntologies(option) : setOntologies,
+              value: ontologies,
+              placeholder: 'Please enter one or more diseases',
+              classNamePrefix: 'select',
+            }),
+          ]),
+
+          RadioButton({
+            value: 'other',
+            defaultChecked: other,
+            onClick: () => {
+              setOther(!other), setHmb(false), setDiseases(false), setGeneral(false);
+            },
+            label: 'Other Use: ',
+            description: 'permitted research use is defined as follows: ',
+            sdsl: true
+          }),
+
+          textarea({
+            className: 'form-control',
+            onBlur: (e) => setOtherText(e.target.value),
+            maxLength: '512',
+            rows: '2',
+            required: other,
+            disabled: !other,
+            placeholder: 'Please specify if selected (max. 512 characters)',
+          }),
+        ]),
+      ]),
+
+      div({className: 'form-group', style: {marginTop: '2rem'}}, [
+        label({style: Styles.MEDIUM}, [
+          '2. Choose any additional constraints you need to put on future uses of your data', br(),
+          span({style: Styles.MEDIUM_DESCRIPTION}, ['Then if necessary, you may choose additional terms on your study\'s data to govern it\'s use by adding requirements or limitations.']),
+        ]),
+        div({}, [
+          RadioButton({
```

Comment:
I believe all section 2 questions should be checkboxes. Using radio buttons to simulate checkboxes is an anti-pattern: https://www.nngroup.com/articles/checkboxes-vs-radio-buttons/